Amplify/Categories/DataStore/Model/Internal/Schema/ModelSchema+Identifiers.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ModelSchema+Identifiers.swift
+//  
+//
+//  Created by Sapir Muallem on 23/04/2023.
+//
```

Comment:
Hi, @sapirmuallem , could you please change it to match the rest of the headers in the project?

```
//
// Copyright Amazon.com Inc. or its affiliates.
// All Rights Reserved.
//
// SPDX-License-Identifier: Apache-2.0
//
```